Lib/tokenize.py

```diff
@@ -34,6 +34,7 @@
 import sys
 from token import *
 from token import EXACT_TOKEN_TYPES
+import _tokenize as c_tokenizer
```

Comment:
Maybe reexport this as `_c_tokenizer`. Otherwise we might encourage usage of the c_tokenizer directly? Are we okay with that?

Reply:
Ah good point!